src/action/common/create_users_and_groups.rs

```diff
@@ -243,6 +281,11 @@ impl Action for CreateUsersAndGroups {
             }
         }
 
+        // We don't actually need to do this, when a user is deleted they are removed from groups
```

Comment:
Do we want to leave this commented code hanging around?

Reply:
I kinda do, so that later when I go "why is this missing?" I see the comment and go "Ah ha!"